packages/scoped-elements/README.md

```diff
@@ -8,32 +8,38 @@ tags:
 
 # Scoped elements
 
-[//]: # 'AUTO INSERT HEADER PREPUBLISH'
-
 Scope element tag names avoiding naming collision and allowing to use different versions of the same web component in your code.
 
 ## Installation
 
 ```bash
-npm i --save @open-wc/scoped-elements
+npm i --save @open-wc/scoped-elements @webcomponents/scoped-custom-element-registry
```

Comment:
Is this duplicated?

Reply:
yeah, we have all the docs duplicate right now... so they also appear on NPM... will change that for all packages in a later PR